src/python/grpcio_tests/tests_aio/unit/timeout_test.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2020 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests behavior of the timeout mechanism on client side."""
+
+import asyncio
+import logging
+import platform
+import random
+import unittest
+import datetime
+
+import grpc
+from grpc.experimental import aio
+
+from tests_aio.unit._test_base import AioTestBase
+from tests_aio.unit import _common
+
+_SLEEP_TIME_UNIT_S = datetime.timedelta(seconds=1).total_seconds()
+
+_TEST_SLEEPY_UNARY_UNARY = '/test/Test/SleepyUnaryUnary'
+_TEST_SLEEPY_UNARY_STREAM = '/test/Test/SleepyUnaryStream'
+_TEST_SLEEPY_STREAM_UNARY = '/test/Test/SleepyStreamUnary'
+_TEST_SLEEPY_STREAM_STREAM = '/test/Test/SleepyStreamStream'
+
+_REQUEST = b'\x00\x00\x00'
+_RESPONSE = b'\x01\x01\x01'
+
+# logging.getLogger('asyncio').setLevel(logging.WARNING)
+
+class _GenericHandler(grpc.GenericRpcHandler):
+
+    def __init__(self):
+        self._routing_table = {
```

Comment:
Nit: Any reason for this to be an instance member rather than a class member?

Reply:
It can be both, but I didn't find a nicer way to initiate those values. Comparing:
1. Using `__init__`, like this;
2. Using `__new__`, feels a bit too powerful;
3. Using class constant leads to "NameError: name '_GenericHandler' is not defined".

Suggestions welcomed.

Reply:
Option 3 was what I was expecting. I don't see a reference to `_GenericHandler` in this statement, so I'm surprised by that error.

Reply:
In snippet below, if the class constants is referencing the methods, it will generate an `NameError`. 

```python
class A:
    val = A.func_1(1) 

    @staticmethod
    def func_1(x):
        return x*2
```

After review this trunk of code, you are right, we could simplify it. The routing table is now module constants, since those server method handlers are not using `self`.